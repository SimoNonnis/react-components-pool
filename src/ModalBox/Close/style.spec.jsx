/* eslint-env jest */
import React from 'react';

import theme from 'themes';
import CloseModal from './style';

describe('Test <CloseModal />', () => {
  it('should render CloseModal correctly', () => {
    const wrapper = shallow(<CloseModal theme={theme} />);
    expect(wrapper).toMatchSnapshot();
  });
});
