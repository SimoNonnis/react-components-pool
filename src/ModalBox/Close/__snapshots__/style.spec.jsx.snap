// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test <CloseModal /> should render CloseModal correctly 1`] = `
.c0 {
  -webkit-appearance: none;
  -moz-appearance: none;
  appearance: none;
  background-color: transparent;
  border: 0;
  color: #1D3141;
  font-family: "proxima-nova","Open Sans","Helvetica Neue",Helvetica,Arial,sans-serif;
  font-size: 43px;
  font-weight: 300;
  line-height: 25px;
  padding: 5px;
  position: absolute;
  right: 0;
  top: 0;
  z-index: 4;
}

.c0:hover {
  cursor: pointer;
}

.c0:active {
  outline: none;
}

@media (min-width:1050px) {
  .c0 {
    right: 3px;
    top: 3px;
  }
}

<button
  className="c0"
/>
`;
